include/SDL3/SDL_time.h

```diff
@@ -0,0 +1,244 @@
+freely, subject to the following restrictions:
+
+1. The origin of this software must not be misrepresented; you must not
+   claim that you wrote the original software. If you use this software
+   in a product, an acknowledgment in the product documentation would be
+   appreciated but is not required.
+2. Altered source versions must be plainly marked as such, and must not be
+   misrepresented as being the original software.
+3. This notice may not be removed or altered from any source distribution.
+*/
+
+#ifndef SDL_time_h_
+#define SDL_time_h_
+
+/**
+ *  \file SDL_time.h
+ *
+ *  Header for the SDL realtime clock and date/time routines.
+ */
+
+#include <SDL3/SDL_error.h>
+#include <SDL3/SDL_stdinc.h>
+
+#include <SDL3/SDL_begin_code.h>
+/* Set up for C function definitions, even when using C++ */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ *
+ * A structure holding a calendar date and time broken down into its components.
+ *
+ * \sa SDL_GetRealtimeClock
+ * \sa SDL_TimeSpecToUTCDateTime
+ * \sa SDL_TimeSpecToLocalDateTime
+ * \sa SDL_DateTimeToTimeSpec
+ * \sa SDL_FileTimeToUTCDateTime
+ * \sa SDL_FileTimeToLocalDateTime
+ */
+typedef struct SDL_DateTime
+{
+    int year;                  /**< Year */
+    int month;                 /**< Month [01-12] */
+    int day;                   /**< Day of the month [01-31] */
+    int hour;                  /**< Hour [0-23] */
+    int minute;                /**< Minute [0-59] */
+    int second;                /**< Seconds [0-60] */
+    int nanosecond;            /**< Nanoseconds [0-999999999] */
+    int day_of_week;           /**< Day of the week [0-6] (0 being Sunday) */
+    int utc_offset;            /**< Seconds east of UTC */
+} SDL_DateTime;
+
+typedef enum SDL_DAY_OF_WEEK
+{
+    SDL_DAY_OF_WEEK_SUNDAY = 0,
+    SDL_DAY_OF_WEEK_MONDAY = 1,
+    SDL_DAY_OF_WEEK_TUESDAY = 2,
+    SDL_DAY_OF_WEEK_WEDNESDAY = 3,
+    SDL_DAY_OF_WEEK_THURSDAY = 4,
+    SDL_DAY_OF_WEEK_FRIDAY = 5,
+    SDL_DAY_OF_WEEK_SATURDAY = 6
+} SDL_DAY_OF_WEEK;
+
+/**
+ *
+ * The preferred date format of the current system locale.
+ *
+ * \sa SDL_GetDateTimeLocalePreferences
+ */
+typedef enum SDL_DATE_FORMAT
+{
+    SDL_DATE_FORMAT_YYYYMMDD = 0, /**< year/month/day */
+    SDL_DATE_FORMAT_DDMMYYYY = 1, /**< day/month/year */
+    SDL_DATE_FORMAT_MMDDYYYY = 2, /**< month/day/year */
+} SDL_DATE_FORMAT;
+
+/**
+ *
+ * The preferred time format of the current system locale.
+ *
+ * \sa SDL_GetDateTimeLocalePreferences
+ */
+typedef enum SDL_TIME_FORMAT
+{
+    SDL_TIME_FORMAT_24HR = 0, /**< 24 hour time */
+    SDL_TIME_FORMAT_12HR = 1, /**< 12 hour time */
+} SDL_TIME_FORMAT;
+
+/**
+ *  Global date/time properties
+ *
+ *  - `SDL_PROP_GLOBAL_SYSTEM_DATE_FORMAT_NUMBER`: the index of the SDL_DATE_FORMAT enum entry with the
+ *    preferred date display format for the system locale.
+ *  - `SDL_PROP_GLOBAL_SYSTEM_TIME_FORMAT_NUMBER`: the index of the SDL_TIME_FORMAT enum entry with the
```

Comment:
Suggest: "the SDL_TIME_FORMAT to use as the preferred..."
This should compile without a cast in SDL_SetNumberProperty()

Reply:
Done